echopype/convert/set_groups_ek80.py

```diff
@@ -659,6 +660,11 @@ def merge_save(ds_combine, ds_type, group_name):
         else:
             ds_beam = merge_save(ds_power, "power", group_name="/Sonar/Beam_group1")
 
+        # manipulate Datasets to adhere to convention
```

Comment:
I suggest a more fleshed out comment. Otherwise it sounds generic. Say:

"Manipulate some Dataset dimensions to adhere to convention"